### ISSUE TITLE
Subpar results compared to Mask-RCNN

### ISSUE THREAD

**Issue by 4ri4Slr** [mrlooi/maskrcnn-benchmark#47]
## ❓ Questions and Help
Hi there,

I know this repo is not maintained anymore, but I have a question and hope I could use some insights. 

After a few modifications, I was able to run the code using Pytorch 1.1 + CUDA 11.1. I trained the network for 2500 iterations on a custom dataset consisting of a few dozen of images and tested it on a sample batch from the training set. The segmentation results are far less accurate compared to the original Mask-RCNN benchmark even the training loss values are somewhat identical in both training regimes:

**Rotated Mask-RCNN Losses:**  iter: 2500  loss: 0.3896 (0.5723)  loss_classifier: 0.1263 (0.1936)  loss_box_reg: 0.0635 (0.1156)  loss_mask: 0.1996 (0.2631) 

**Mask-RCNN Losses**:  iter: 2500  loss: 0.3355 (0.7216)  loss_classifier: 0.0923 (0.2032)  loss_box_reg: 0.0852 (0.1953)  loss_mask: 0.1208 (0.1620)  

The only warning I get during training rotated mask-rcnn is the following, which I have not been able to traceback completely: 

```
/content/rotated/rotated_maskrcnn/maskrcnn_benchmark/structures/bounding_box.py:268: UserWarning: indexing with dtype torch.uint8 is now deprecated, please use a dtype torch.bool instead. (Triggered internally at  ../aten/src/ATen/native/IndexingUtils.h:30.)
  bbox = BoxList(self.bbox[item], self.size, self.mode)
```

Is the performance drop an expected behaviour when the network is trained for the same number of iterations as the original mask rcnn, or is it because of a bug in interpreting the bounding boxes caused by a different PyTorch version than that suggested in the repo?

Thank you,
